zone/client.cpp

```diff
@@ -11647,6 +11648,14 @@ void Client::RegisterBug(BugReport_Struct* r) {
 	b.bug_report          = r->bug_report;
 	b.system_info         = r->system_info;
 
+#ifdef LUA_EQEMU
+	bool ignoreDefault = false;
+	LuaParser::Instance()->RegisterBug(this, b, ignoreDefault);
+	if (ignoreDefault) {
+		return;
+	}
+#endif
+
```

Comment:
I know the old stuff doesn't use `snake_case` but can we use it here?

Reply:
Fixed